pom.xml

```diff
@@ -61,6 +61,20 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
the only module that does not need them because its a server-specific module is the management-entity-server module. and management-entity-test module. Otherwise all other modules are susceptible to be used externally. I just wanted to avoid plenty of copy paste and the eventuality that we miss some osgi generation on day that would force us to do another release just because of that.
So there's some pros and cons both ways.